### DISCUSSION TITLE
Nuclei template option not working on version 2.4.1

### DISCUSSION THREAD

**Discussion by cfsdes** [projectdiscovery/nuclei#876] [General]
Nuclei option -t it's converting a string to lowercase when setting a specific template:

<img width="1284" alt="Captura de Tela 2021-07-28 às 00 04 00" src="https://user-images.githubusercontent.com/20214824/127257221-8d247759-5193-47e0-bcc4-1e92bc5f4bcf.png">

**Comment by ehsandeep** [comment]
@cfsdes this is fixed in the latest release of nuclei
